cmd/sqlcmd/sqlcmd.go

```diff
@@ -117,25 +118,140 @@ func (a SQLCmdArguments) authenticationMethod(hasPassword bool) string {
 }
 
 func Execute(version string) {
-	ctx := kong.Parse(&args, kong.NoDefaultHelp())
-	if args.Version {
-		ctx.Printf("%v", version)
-		os.Exit(0)
+	// Cobra Tryout #2 : This has version and help flags. But they are also getting stucked inside run function without error
+	rootCmd := &cobra.Command{
+		Use:   "testCommand",
+		Short: "A brief description of your command",
+		Run: func(cmd *cobra.Command, argss []string) {
+			vars := sqlcmd.InitializeVariables(!args.DisableCmdAndWarn)
+			setVars(vars, &args)
+
+			if args.Version {
+				fmt.Printf("%v\n", version)
+				os.Exit(0)
+			}
+
+			if args.Help {
+				fmt.Print(cmd.UsageString())
+				os.Exit(0)
+			}
+
+			exitCode, err := run(vars, &args)
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
```

Comment:
we don't need to print  the error from run because run already prints relevant errors.